CoffeeMachine.java

```diff
@@ -0,0 +1,90 @@
+package Lesson_12;
+
+import java.util.Scanner;
+
+public class CoffeeMachine {
+    private String esspresso = "Esspresso";
+    private String macchiato = "Macchiato";
+    private String cappuccino = "Cappuccino";
+    private String milk = "Milk";
+    private String hotChocolate = "Hot Chocolate";
+    private String drinkChoice;
+    private int answer;
+    private char choice;
+    private String[] drinkList = {"Espresso", "Macchiato", "Cappuccino"};
+
+    public CoffeeMachine(){
+        this.esspresso = esspresso;
+        this.macchiato = macchiato;
+        this.cappuccino = cappuccino;
+        this.milk = milk;
+        this.hotChocolate = hotChocolate;
+        this.drinkChoice = drinkChoice;
+        this.answer = answer;
+        this.drinkList=drinkList;
+        this.choice=choice;
+    }
+
+    public void getDrink(){
+        System.out.println("Choose your drink");
+        int n = 0;
+        for (String menu:this.drinkList){
+            System.out.println(++n+ " | " + menu);
+        }
+        makeChoice();
+    }
+
+    private void makeChoice(){
+        Scanner reader = new Scanner(System.in);
+        this.answer = reader.nextInt();
+
+        if (this.answer==1){
+            addSugar();
+            System.out.println("Your " + this.esspresso + " is now ready." +  "\n-------------------------");
+        }else if (this.answer==2){
+            addSugar();
+            System.out.println("Your " + this.macchiato + " is now ready." + "\n-------------------------");
+        }else if (this.answer==3){
+            addSugar();
+            System.out.println("Your " + this.cappuccino + " is now ready." + "\n-------------------------");
+        }
+        anotherDrink();
+    }
+
+    private void anotherDrink (){
+        Scanner reader = new Scanner(System.in);
+
+        do{
+            System.out.println("Do you want to make another drink? y/n");
+            this.choice = reader.next().toLowerCase().charAt(0);
+            if(this.choice=='y'){
```

Comment:
Generally, good code practice is to have spaces between keywords in the code. 
For example, this if statement should be: 
```
if (this.choice == 'y' ) { 
```
This should be fixed generally throughout whole code base